### PR TITLE
tabula-HOME AFFAIRS - DETAIL COVID 19 PROCUREMENT.csv

### PR DESCRIPTION
**Pull request by rmaclean**
#63

image
image

### CODE REVIEW

**Review by martink-rsa — CHANGES_REQUESTED** [review]
Hi @rmaclean ,

Thank you for the hard work that you have done.

We can't merge this file at this stage due to issues with the data. Many of the fields don't match the data from the source. Here is an example:

image

2. There appears to be an issue with one of the lines that is preventing the file from being parsed by .csv readers, such as the Github tool to display the file. 

image

These changes will need to be done before the file can be merged.

**Comment by tristynferreiro** [comment]
I fixed it

**Pull request merged**
